test/specs/empirestatebuilding.spec.js

```diff
@@ -0,0 +1,33 @@
+import EmpireStateBuildingPage from "../pageobjects/empirestatebuilding.page";
+
+describe("Wikipedia empire state building page", () => {
+    it("verify  the link next to 'Preceded by' would take you to the page for the Chrysler Building", () => {
+        browser.url("https://en.wikipedia.org/wiki/Empire_State_Building");
+        EmpireStateBuildingPage.waitPageToLoad();
+        let expected = "/wiki/Chrysler_Building";
+        let actual = EmpireStateBuildingPage.precededByLink.getAttribute(
+            "href"
+        );
+        expect(actual).toEqual(expected);
+    });
+
+    it("verify the link next to 'Surpassed by' would take you to the page for the World Trade Center (North tower)", () => {
+        let expected = encodeURI("/wiki/World_Trade_Center_(1973–2001)");
+        let actual = EmpireStateBuildingPage.surpassedByLink.getAttribute(
+            "href"
+        );
+        expect(actual).toEqual(expected);
+    });
+
+    it(`Each photo thumbnail on the page should have an icon in the caption below it (<div class="magnify">).`, () => {
```

Comment:
Since I turned the Extend repo into a template repo, I can't actually run your code now. Does this test pass?

Reply:
@m-schrepel  please find the result of local run
![image](https://user-images.githubusercontent.com/12701069/98731473-a2c45b80-2352-11eb-83d6-ef64394ed0f4.png)

Reply:
I see. This test _should_ be failing. There is a thumbnail on the page that does not have a magnify icon.

Reply:
`$('.thumb').size() === $('.thumb .magnify').size()`

If you run this on the page itself, you'll get `false`

Reply:
Yes there is one image not having magnify icon. However, if you see that image's class name is not set as thumborder not thumbimage. The rest of images will have their class name as thumbimage. Therefore, in my locator, i only locate all images whose class name is thimbimage as below:
  "//*[img[@class='thumbimage']]/following-sibling::*[@class='thumbcaption']/*[1]"